index.html

```diff
@@ -52,6 +52,164 @@ <h1>I'm Arturo Alvarez.Glad to see you!</h1>
+						</div>
+						<ul class="technologies">
+							<li>HTML</li>
+							<li>CSS</li>
+							<li>JavaScript</li>
+						</ul>
+						<a href="https://tuhogaraunclic.com" class="see">See project</a>
+					</div>
+				</div>
+				<div class="card">
+					<div class="image-container background-light">
+						<img src="/assets/elrincon.png" alt="El Rincon">
+					</div>
+					<div class="info">
+						<div class="info-title">
+							El Rincon
+						</div>
+						<div class="canopy">
+							CANOPY
+							<ul>
+								<li>Back End Dev</li>
+								<li>2018</li>
+							</ul>
+						</div>
+						<div class="extract">
+							<p>Web page design for the menu, orders and financial administration</p>
+						</div>
+						<ul class="technologies">
+							<li>HTML</li>
+							<li>CSS</li>
+							<li>JavaScript</li>
+						</ul>
+						<a href="https://restaurante-web-eta.vercel.app/" class="see">See project</a>
+					</div>
+				</div>
+				<div class="card">
+					<div class="image-container background-dark">
+						<img src="/assets/farmaciajesus.png" alt="Farmacia Jesus">
+					</div>
+					<div class="info">
+						<div class="info-title">
+							Farmacia Jesus
+						</div>
+						<div class="canopy">
+							CANOPY
+							<ul>
+								<li>Back End Dev</li>
+								<li>2018</li>
+							</ul>
+						</div>
+						<div class="extract">
+							<p>Drug store landing page to help clients to find the best prices for their medicine</p>
+						</div>
+						<ul class="technologies">
+							<li>HTML</li>
+							<li>CSS</li>
+							<li>JavaScript</li>
+						</ul>
+						<a href="https://farmaciajesus.onrender.com" class="see">See project</a>
+					</div>
+				</div>
+			</div>
+		</section>
+		<section class="about">
+			<div class="title">
+				<p>About Myself</p>
+			</div>
+			<div class="me position">
+					<p>
+							Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+					</p>
+			</div>
+			<div class="contact position">
+				<p>LET'S CONNECT</p>
+				<ul class="social width">
+					<li>
+						<a href="https://twitter.com/" class="a">
+							<img src="/assets/twitter.svg" alt="Twitter logo">
+						</a>
+					</li>
+					<li>
+						<a href="https://www.linkedin.com/in/arturo-alvarez-martinez-ab9b77151/" class="a">
+							<img src="/assets/Linkedin icon.svg" alt="Linkedin logo">
+						</a>
+					</li>
+					<li>
+						<a href="https://github.com/Artufelii" class="a">
+							<img src="/assets/Vector.svg" alt="Github logo">
+						</a>
+					</li>
+				</ul>
+				<a href="/assets/ArturoAlvarez-Resume.pdf" download="ArturoAlvarez-Resume" class="see">Get my resume</a>
+			</div>
+			<div class="skills">
+				<div class="skill">
+					<div class="name">
+						<p>Languages</p>
```

Comment:
The technology categories ("Languages", "Frameworks" and "Skills") needs to be in a ul, each of them in a li. Kindly fix them to make your web page more semantic.